app/routes/pages/components/LandingPage.css

```diff
@@ -0,0 +1,135 @@
+@import 'app/styles/variables.css';
+
+.pageContainer {
+  position: relative;
+  padding: 1em;
+  margin-top: -2em;
+}
+
+.banner {
+  width: 100%;
+  max-width: 55rem;
+  display: block;
+  margin: 1rem auto;
+  object-fit: scale-down;
+}
+
+.statisticsContainer {
+  display: flex;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-bottom: 2.5em;
+  margin-top: 1em;
+}
+
+.whoWhatWhyContainer {
+  display: flex;
+}
+
+.locationContainer {
+  display: flex;
+}
+
+.emailContainer {
+  display: flex;
+  margin-top: -2em;
+}
+
+.committeeEmails {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.organizationContainer {
+  display: flex;
+}
+
```

Comment:
`.statisticsContainer`
`.whoWhatWhyContainer`
`.locationContainer`
`.emailContainer`
`.committeeEmails`
`.organizationContainer`

Just import { Flex } and use that in the `.js` file. You don't need classes for them.